app/kubemci/pkg/gcp/sslcert/sslcertsyncer.go

```diff
@@ -114,6 +124,13 @@ func (s *SSLCertSyncer) updateSSLCert(desiredCert *compute.SslCertificate) (stri
 	name := desiredCert.Name
 	fmt.Println("Deleting existing ssl cert", name, "and recreating it to match the desired state.")
 	// SSL Cert does not support update. We need to delete and then create again.
+	// Note: This is different than what we do in ingress-gce.
+	// In ingress-gce, we first create a new cert with a different name,
+	// update the target proxy to point to the new cert and then delete
+	// the old cert.
+	// What we do here is simpler, but can lead to downtime in the brief period
+	// when we have deleted the old cert but havent created the new one.
+	// TODO(nikhiljindal): Converge this with ingress-gce by sharing the same code.
```

Comment:
file a bug for this?

Reply:
Done https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/issues/124